projects/ngneat/helipopper/src/lib/helipopper.directive.ts

```diff
@@ -118,6 +118,8 @@ export class HelipopperDirective implements OnDestroy {
   }
 
   @Output() helipopperClose = new Subject();
+  @Output() helipopperShowing = new Subject<boolean>();
```

Comment:
Can we change it to `helipopperVisible` please

Reply:
Done, looking forward for the next message about readme.